src/frontend/src/lib/components/tokens/TokenCard.svelte

```diff
@@ -2,12 +2,23 @@
 	import Card from '$lib/components/ui/Card.svelte';
 	import TokenLogo from '$lib/components/tokens/TokenLogo.svelte';
 	import type { Token } from '$lib/types/token';
+	import Tag from '$lib/components/ui/Tag.svelte';
+	import { nonNullish } from '@dfinity/utils';
 
 	export let token: Token;
+
+	const ariaLabel = nonNullish(token.displayName)
```

Comment:
isn't the `ariaLabel` just equals to the `name`?

Reply:
ah! it really depends on what we would like: as I have written it, if we display for example `<Tag>ck</Tag> Tether USD`, the arialabel is "ckTether USD", otherwise it would be "ckUSDT"

what do you think? what is better?

Reply:
I was thinking that we should someday actually use zod for the types and the displayName.prefix + displayName.name === token.name

Reply:
eh, but in the ck-tokens' case it will be `displayName.prefix + displayName.name !== token.name` for now

P.S. agreed on Zod, but I imagine this is another refactoring in a separate PR

Reply:
> displayName.prefix + displayName.name !== token.name

Why? The token name is the token name, period. This is given by its related ledger. Really not sure what we are doing here then?

> agreed on Zod

Definitely for the future, there is even a Jira task I think.

Reply:
Basically they asked that for ck-tokens, instead of having the normal name, that is "ckXXXX", they want ck + the name of the twin token. For example, for ckUSDT, instead of showing "ckUSDT" they want to see "ck" + "Tether USD", for "ckBTC" show "ck Bitcoin" (or "ckBitcoin")

At least, this is the last version they agreed upon.

Plus they liked the "ck" to be in a tag format

Reply:
so here the name is `ckLINK`.

Reply:
In other words, if one want to have a different name, then a proposal should be made to update the tokens name. 

Like you would not display "dallar" instead of "dollar" because all other platforms in the word display "dollar".

Reply:
yes, I think they are looking into that, but they are not sure they want to change the official name of the token. So, for now, they saw the mockup of this new style and they liked it, even with the wrong name. But as you said, they will probably change it again later on. I think they want to see iterations of it